sdk/src/Core/Amazon.Runtime/EventStreams/Internal/EventStream.cs

```diff
@@ -253,22 +262,39 @@ protected void Process()
         {
 #if AWS_ASYNC_API
             // Task only exists in framework 4.5 and up, and Standard.
-            Task.Run(() => ProcessLoop());
+            _ = ProcessLoopAsync();
```

Comment:
You still want to wrap this async version with a Task.Run. Otherwise, the caller is still blocked until the first actual IO bound operation happens, which is deep down in the network stream read.

Reply:
Good call on adding the `Task.Run`. Added that back in.